pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -107,8 +143,8 @@ func NewDiscoveryServer(env *model.Environment, generator core.ConfigGenerator)
 
 	out.DebugConfigs = os.Getenv("PILOT_DEBUG_ADSZ_CONFIG") == "1"
 
-	pushThrottle := intEnv("PILOT_PUSH_THROTTLE", 25)
-	pushBurst := intEnv("PILOT_PUSH_BURST", 100)
+	pushThrottle := intEnv("PILOT_PUSH_THROTTLE", 100)
+	pushBurst := intEnv("PILOT_PUSH_BURST", 250)
```

Comment:
?? Why change these defaults? Also whats the impact? These have no units.. So, it already feels arbitrary in terms of the 25/100. Bumping one by 4x and another by 2.5x makes this even more strange..

Reply:
Because with this optimization we can push faster. Just like the original values, it's based on testing
( with 1000 connected endpoints )